microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -188,17 +370,36 @@ See also:
 
 Use this API to publish the current draft of the product to the Microsoft Edge Add-ons website.
 
+##### [v1.1](#tab/v1-1)
+
+<!-- todo: update for 1.1 -->
```

Comment:
h2 # 3: update 2 code blocks
@yvsubhash 
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3269&tabs=v1#publishing-the-submission

Reply:
v1.1 code in tabs is ready for review.  closing.